support-frontend/app/controllers/CreateSubscriptionController.scala

```diff
@@ -103,7 +103,7 @@ class CreateSubscriptionController(
         // For Stripe payments, recaptcha validation is done earlier in the flow
         val recaptchaToken =
           request.body.paymentFields match {
-            case Left(dd: DirectDebitPaymentFields) => Some(dd.recaptchaToken)
+            case dd: DirectDebitPaymentFields => Some(dd.recaptchaToken)
```

Comment:
Previously the `paymentFields` value was an `Either` which could be payment field data or redemption data. Now it will always be of type `PaymentFields` which simplifies the code significantly.